SDKGenerator.njsproj

```diff
@@ -989,6 +989,12 @@
     <None Include="targets\javascript\testingFiles\PlayFabApiTest.ts" />
   </ItemGroup>
   <ItemGroup>
+    <TypeScriptCompile Include="generate-plugins.ts">
+      <SubType>Code</SubType>
+    </TypeScriptCompile>
+    <TypeScriptCompile Include="generate-sdk.ts">
+      <SubType>Code</SubType>
+    </TypeScriptCompile>
```

Comment:
Visual Studio occasionally does excessive tagging of these files in the proj file for some reason. I don't know why it does this vs the format below, and how to trigger one or the other.
I usually manually convert it to the single-line format when it happens.
I'm willing to start ignoring it if we decide I'm being too OCD about it though.

Reply:
I will update them manually to make one-liners as below, good point. I like minimality, too :)

---
In reply to: [198236509](https://github.com/PlayFab/SDKGenerator/pull/142#discussion_r198236509) [](ancestors = 198236509)